draft-foudil-securitytxt.md

```diff
@@ -453,26 +466,105 @@ unstructured     = <imported from section 3.2.5 of [RFC5322]>
 
 "ext-field" refers to extension fields, which are discussed in {{extensibility}}
 
-# Security considerations
+# Security Considerations
+
+## Compromised Files and Redirects
+
+An attacker that has compromised a website is able to compromise
+the "security.txt" file as well or setup a redirect to their own site.
+This can result in security reports not being received by the organization
+or sent to the attacker.
+
+To protect against this, organizations SHOULD digitally sign their "security.txt"
+files (as per {{signature}}), use the canonical directive to sign the location
+of the file (as per {{canonical}}), and regularly monitor the file and
+the referenced resources to detect tampering.
+
+Security researchers SHOULD check the "security.txt" including verifying
+the digital signature and checking historical records before using the information
```

Comment:
By "historical records", do you mean stuff like the Wayback machine? Or records that the security researcher is keeping themselves?

Reply:
either